js_utils/text_validation.js

```diff
@@ -69,6 +69,10 @@ export class TextValidation {
   validate_plate_barcode(text) {
     // refactored for 100% code coverage on JEST Unit testcases.
     let response = " "; // let us first assume that incoming text is invalid unless it passes the criteria.
+    // As the validation rules are on Vuex data of barcode and this can be null a check is required.
+    if (text == null) {
+      return response;
+    }
```

Comment:
Line 83 is not covered by any tests, please create a test to confirm that behaves as expected.  Line 110 is always True--please create a test where everything is valid except the year and confirm the code behaves as expected

Reply:
Agree to add a `null`, `MBFF0440991` and `MB2204409913` included now its 100% coverage with all scenarios verified.